react/src/index.tsx

```diff
@@ -38,14 +41,35 @@ const theme = extendTheme({
 
 const AuthenticationProtectedApp = withAuthenticationRequired(App);
 
+const SentryProfiledApp = Sentry.withProfiler(AuthenticationProtectedApp);
+
+const sentryDsn = process.env.REACT_APP_SENTRY_DSN;
```

Comment:
Do we have to distinguish between two projects (FE and BE) in Sentry for the new app? Otherwise you can use `SENTRY_DSN` which is already set as env variable in CircleCI (the name is dictated by the Sentry SDK). See also #303 

Also for environment I introduced the env variable `SENTRY_ENVIRONMENT` with possible values staging, demo, production.

Reply:
I think it makes sense to separate the two systems also in different projects. 
Will integrate the `SENTRY_ENVIRONMENT` env variable! :)

Reply:
@vahidbazzaz @HaGuesto I don't have required permissions, can you rename the existing project `boxwise-production` into `boxtribute2-backend` and create a new project `boxtribute2-frontend` in Sentry? Or you grant me permissions to do it myself :wink: